src/classes/Client.ts

```diff
@@ -80,8 +80,18 @@ export default class Client {
 	}
 
 	/** Get playlist information and its videos by playlist id or URL */
-	async getPlaylist(playlistIdOrUrl: string): Promise<Playlist | undefined> {
+	async getPlaylist(playlistIdOrUrl: string): Promise<Playlist | MixPlaylist | undefined > {
```

Comment:
Please use generic type for the return type (like [.getVideo()](https://github.com/SuspiciousLookingOwl/youtubei/blob/main/src/classes/Client.ts#L97))

```ts
async getPlaylist<T extends Playlist | MixPlaylist | undefined>(playlistIdOrUrl: string): Promise<T> 
```